### ISSUE TITLE
What about a prebuilt binaries repo?

### ISSUE THREAD

**Issue by 3oheicrw** [brechtsanders/winlibs_recipes#4]
Instead of requiring us to download, setup and build everything ourselves. What about you having a build server that offer prebuilt binaries? So it will be very similar to MSYS2's pacman. The facility needed for this should be already available on winlibs_tools isn't it?

**Comment by brechtsanders** [comment]
Yes, that is what https://github.com/brechtsanders/winlibs_tools/ and https://github.com/brechtsanders/winlibs_recipes are for. I'm still working on some issues to get this up to production quality, but the good news is I'm already able to build over 2500 software packages.

**Comment by 3oheicrw** [comment]
> Yes, that is what https://github.com/brechtsanders/winlibs_tools/ and https://github.com/brechtsanders/winlibs_recipes are for. I'm still working on some issues to get this up to production quality, but the good news is I'm already able to build over 2500 software packages.

Well, it's awesome. You could get featherpad to build on Windows? I saw it on stable recipes. The developer of it is very arrogant. It's more than two times I asked him to add Windows support and both of the time he dismissed it then his worshipers idolizing him go and bash me (and Windows, too). So I told myself to never do this again, as he doesn't care about us. I don't have the knowledge to do that myself but if I can I would already forked it. BTW, I saw qtwebkit and qtwebengine on the working recipes. Man, if you could get qtwebengine builds with MINGW, it's so awesome. Even though switched to clang on Windows, the Chromium guys (again as much as arrogant as the featherpad developer), require us to install Visual Studio to use the Windows headers from it. Before the switch to clang they used MSVC. MINGW has never been a thing in their minds. About qtwebkit, as there is an package for it on MSYS2 I think it should build just fine with MINGW.

**Comment by 3oheicrw** [comment]
About arrogant developers again. I wanted to tell you that this C++ IDE would build out of the box on MSYS2. The developer of it, arrogant or not I don't know, seems only answer issues submitted in Chinese and he dismissed all issues submitted by English. It seems it's development mainly on Chinese's gitee.io and github.com is only served as an international portal to them.

https://github.com/royqh1979/RedPanda-CPP/issues/61

**Comment by brechtsanders** [comment]
Featherpad builds, but I haven't tested it apart from just starting the application.
Anything WebKit-related I wasn't able to build yet I'm afraid.
Which IDE were you referring to exactly?

**Comment by 3oheicrw** [comment]
> Featherpad builds, but I haven't tested it apart from just starting the application. Anything WebKit-related I wasn't able to build yet I'm afraid. Which IDE were you referring to exactly?

This: https://github.com/royqh1979/RedPanda-CPP

About Webkit, why don't you have a look at how the MSYS2 packagers do it? I'm sure there is a qtwebkit package:

https://packages.msys2.org/package/mingw-w64-x86_64-qtwebkit?repo=mingw64

But don't know how to find the patches they used.

**Comment by brechtsanders** [comment]
Just had a look at RedPanda-CPP and it's already got everything on board to build on Windows.

You can find my recipe [here](https://github.com/brechtsanders/winlibs_recipes/blob/main/recipes/redpanda-cpp.winlib) and it builds without modification.

About WebKit: it always comes down to the missing dependency of corefoundation/WTF. If I can figure out how to get that I believe it should be possible to build.
